src/main/java/seedu/foodiebot/model/randomize/Randomize.java

```diff
@@ -61,11 +70,37 @@ public void getSelectedCanteen(FileReader file) {
         }
     }
 
-    //public void parseCanteenObject (JSONObject canteen) {
-    //    JSONObject canteenObject = (JSONObject) canteen.get("canteen");
-    //    String canteenName = (String) canteenObject.get("name");
-    //    System.out.println(canteenName);
-    //    canteens.add(canteenName);
-    //}
+    /**
+     * This method produce a list of options from the jsonfile (foodiebot-stalls.json).
+     * @param file which is provided from the RandomizeCommand.
+     */
+    public void getOptions(FileReader file) {
+        int listSize;
+        int index;
+        int i = 0;
+        JSONParser parser = new JSONParser();
+        try {
+            JSONObject obj = (JSONObject) parser.parse(file);
+            JSONArray stallList = (JSONArray) obj.get("stalls");
+            //System.out.println((canteenList));
+
+            listSize = stallList.size();
+
+            while (i < 5) {
+                index = rand.nextInt(listSize);
+                JSONObject stall = (JSONObject) stallList.get(index);
+                selectedStall = (String) stall.get("name");
+                selectedCanteen = (String) stall.get("canteenName");
+                if (!output.contains(selectedStall)) {
+                    output = output + String.format("%d. %s: %s\n", i + 1, selectedCanteen, selectedStall);
+                    i++;
+                }
+            }
+        } catch (IOException | ParseException e) {
+            e.printStackTrace();
```

Comment:
I think you should throw a new `CommandException` here instead of using `e.printStackTrace()` after catching either `IOException` or `ParseException` so that it shows up in the feedback box of the app.